### PR TITLE
update the path on push to include Chart.yaml because the global.yaml…

### PR DESCRIPTION
**Pull request by moosecanswim**
last update for helm-docs.  I forgot to include chart.yaml as a trigger for build docs.  The gloabal.yaml doc config info is based on a "fake chart" that pulls in data from fabric, sence, data etc charts.  This will make sure that top level requirements match the actually checked in charts when Chart.yaml are updated.

### CODE REVIEW

File: spire/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 A Helm chart to deploy Spire
 
-![Version: 4.0.2](https://img.shields.io/badge/Version-4.0.2-informational?style=flat-square) ![AppVersion: 1.6.0](https://img.shields.io/badge/AppVersion-1.6.0-informational?style=flat-square)
+![Version: 4.0.4](https://img.shields.io/badge/Version-4.0.4-informational?style=flat-square) ![AppVersion: 1.6.0](https://img.shields.io/badge/AppVersion-1.6.0-informational?style=flat-square)
```
Comment (chrisbsmith) [inline spire/README.md:5]: Is this something we'll have to change every time we update?

Comment (moosecanswim) [reply spire/README.md:5]: That was automatically added in :-D

Comment (chrisbsmith) [reply spire/README.md:5]: 👍

**Review by chrisbsmith — APPROVED** [review]
LGTM

**Pull request merged**